### DISCUSSION TITLE
Enhanced Table plugin for Kibana version 7.12

### DISCUSSION THREAD

**Discussion by bouazizrahma** [fbaligand/kibana-enhanced-table#199] [Q&A] [answered]
Dear Fabien,

I have used the enhanced-table plugin with kibana 7.10.
Now, I installed the latest kibana version 7.12, my question :
Is it possible to install the plugin : enhanced-table-1.11.1_7.11.0 on kibana 7.12 ?
or should I wait for the enhanced-table plugin release for kibana 7.12 ? 

Indeed, in my dashboards, I need both the Enhanced Table and the Document Table.

Thank you very much for your help.

Kind regards,
Rahma BOUAZIZ

**Comment by tonykoval** [comment, marked as answer]
https://github.com/fbaligand/kibana-enhanced-table/issues/197
